.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "Dapr Dev Environment",
+	// Update container version when you update dev-container
+	"image": "docker.io/youngp/dapr-dev:0.1.0",
+	// Uncomment if you want to use your local docker image
+	// "dockerFile": "../docker/Dockerfile-dev",
+	"runArgs": [
+		// On Linux, this will prevent new files getting created as root, but you 
+		// may need to update the USER_UID and USER_GID in .devcontainer/Dockerfile
+		// to match your user if not 1000.
+		"-u", "dapr",
+		// Mount kubernetes and minikube config to dev container
+		"--mount", "type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock",
+		"--mount", "type=bind,source=${env:HOME}${env:USERPROFILE}/.kube,target=/home/dapr/.kube-localhost",
+		"--mount", "type=bind,source=${env:HOME}${env:USERPROFILE}/.minikube,target=/home/dapr/.minikube-localhost",
+		"-e", "SYNC_LOCALHOST_KUBECONFIG=true",
+		"--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined",
+		// Comment out if you want to have the network-isolated dev environment.
+		"--net=host"
+	],
+	"settings": {
+		"terminal.integrated.shell.linux": "/bin/bash",
+		"remote.extensionKind": {
+			"ms-azuretools.vscode-docker": "workspace"
```

Comment:
Forcing the extension to `workspace` should no longer be necessary.

Reply:
oh I see. I will update it. thanks for the review.